### PR TITLE
[Android] 2/2 Replace logo and banner boinc drawables to vectors drawables

### PR DESCRIPTION
**Pull request by talregev**
Replace logo and banner boinc drawables to vector drawable.
Fixes: #2468

### CODE REVIEW

**Comment by talregev** [comment]
Screenshot_20200521-223322_BOINC
Screenshot_20200521-223333_BOINC
Screenshot_20200521-224505_BOINC
Screenshot_20200521-224520_BOINC
Screenshot_20200521-224524_BOINC
Screenshot_20200521-224530_BOINC
Screenshot_20200521-224536_BOINC
Screenshot_20200521-224609_BOINC
Screenshot_20200521-224625_BOINC
Screenshot_20200521-224632_BOINC
Screenshot_20200521-224646_BOINC
Screenshot_20200521-224711_BOINC
Screenshot_20200521-225505_BOINC
Screenshot_20200521-235109_BOINC
Screenshot_20200521-235129_BOINC
Screenshot_20200521-235437_Trebuchet

File: android/BOINC/app/src/main/res/drawable/ic_baseline_menu_black.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="48dp" android:tint="@android:color/black"
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/res/drawable/ic_baseline_menu_black.xml:1]: Please add copyright header

File: android/BOINC/app/src/main/res/drawable/ic_baseline_menu_white.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="48dp" android:tint="@android:color/white"
```
Comment (AenBleidd) [inline android/BOINC/app/src/main/res/drawable/ic_baseline_menu_white.xml:1]: Please add copyright header

**Comment by AenBleidd** [comment]
It's not correct icon color:
image

**Comment by talregev** [comment]
> It's not correct icon color:

From the some API and above, all notification small icon are white. It a standart. 
In the picture above you see the Icon as it was before transformation.

**Pull request merged**